workflow-job

```diff
@@ -399,6 +416,43 @@ flows["${sha1}"] = {
             sh "cd dockerfiles/; docker run ${verbose} ${tmpfsOptions} ${allVolumes} ${testNameDefinitions} ${partialBuildEnv} ${images.jamesCompile} ${mergeBranch}"
             testingJava8.success();
 
+            //Deploying Guice Server with Cassandra + RabbitMQ + Swift
```

Comment:
what happens if you try this workflow on the 3.2.x branch for example?

Reply:
I think it fails, I thought about it but it already true for older branches, we *need* JenkinsFile (or to duplicate the workflow otherwise)